velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -245,6 +245,30 @@ TEST_F(DateTimeFunctionsTest, lastDay) {
   EXPECT_EQ(lastDayFunc(std::nullopt), std::nullopt);
 }
 
+TEST_F(DateTimeFunctionsTest, dateFromUnixDate) {
+  const auto dateFromUnixDate = [&](std::optional<int32_t> value) {
+    return evaluateOnce<int32_t>("date_from_unix_date(c0)", value);
+  };
+
+  // Null test
```

Comment:
This is testing the framework. Please, remove.

Reply:
Removed, thanks.